### DISCUSSION TITLE
[Help]: Missing `destroyHeight` from `AutoHeight()` options

### DISCUSSION THREAD

**Discussion by zaaakher** [davidjerleke/embla-carousel#915] [Help] [answered]
### Summary

Hi @davidjerleke, hope you're doing well.

I update dependencies routinely in some of my projects ([Hawa](https://github.com/sikka-software/hawa) in this case) and it seems the latest version of `embla-carousel-auto-height` doesn't have a `destroyHeight` property in the options of `AutoHeight` function.

I couldn't find any mention in the changelog/release descriptions. Was it replaced with something else?

![image](https://github.com/davidjerleke/embla-carousel/assets/46135573/4f28a0c6-ea3b-4f15-8a6f-25e31d228548)

Thank you for the continuous work you do for this wonderful package 🌹

### If applicable, which variants of Embla Carousel are relevant to this question?

- [X] embla-carousel-auto-height

**Comment by zaaakher** [comment]
Ok after a little digging I found the moment it was removed 👇

 https://github.com/davidjerleke/embla-carousel/commit/8eca652ae340c2e38441f4a430edd12fefaec910#diff-9b492d4efffa04663108927b0281a3403b6426079070f8bb9732cc274abdba41

**Reply by davidjerleke** [reply, marked as answer]
Hi @zaaakher,

Nice to hear from you again 🙂. You're right, it was on my todo for `v8.0.0` but somehow I forgot to remove it. Sorry for the inconvenience. The new behavior is much better and works like this:

- It removes the height style when it destroys/unmounts.
- And if there are no additional styles, it removes the inline `style` property from the container. 

No need to set `destroyHeight` anymore. A breaking change not related to a major version doesn't happen often I promise.

Best,
David

**Reply by zaaakher** [reply]
Hey brother 👋,

Sounds excellent, well done as always 👍! 

Don't worry about it, I kinda wanted to put this discussion out there so that other people can find the answer if they ever had the same question.

If you think it's not enough and other devs might create issues regarding the missing `destroyHeights`. Then perhaps we can add a callout in the [docs](https://www.embla-carousel.com/plugins/auto-height/) saying something like:

> [!note]
> As of `v8.0.0` the option `destroyHeight` is deprecated and not needed anymore. See this [discussion](https://github.com/davidjerleke/embla-carousel/discussions/915#discussioncomment-9798171) for more info

Let me know if you'd like that and I will create a PR for it :) 


However, feel free to @ me anytime you need assistance in anything and I will try to do my best. 👌

**Reply by davidjerleke** [reply]
> I kinda wanted to put this discussion out there so that other people can find the answer if they ever had the same question.

@zaaakher that's great and very helpful 👍.

> If you think it's not enough and other devs might create issues regarding the missing destroyHeights. Then perhaps we can add a callout in the [docs](https://www.embla-carousel.com/plugins/auto-height/)

Good idea!. Let's see if this discussion is enough or if it will cause a lot of trouble.

> However, feel free to @ me anytime you need assistance in anything and I will try to do my best. 👌

Thanks for supporting this project! Will do 🙂.
